'''
    예외처리
'''

# print(int("test"))

try:
    # print(int("try"))
    print(int("1234"))
except Exception as e:
    print(e)
finally:
    print("finally")













